week-2/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return numbers[0]; // complete this statement
 }
 
 function last(arr) {
-  return; // complete this statement
+  return names[2]; // complete this statement
```

Comment:
This will return the 3rd character in the string...

Reply:
You'll need to be creative with this one :) how can you know how many characters an array has, and deduce the index of the last character every time?